src/components/Timeline/index.js

```diff
@@ -1,37 +1,60 @@
 import React, { useState } from 'react';
+import { func } from 'prop-types';
 import {
   StyleSheet, View, TouchableOpacity,
 } from 'react-native';
-import { Ionicons } from '@expo/vector-icons'; // eslint-disable-line import/no-extraneous-dependencies
+import { Left, Right } from 'native-base';
+import { Ionicons, MaterialCommunityIcons } from '@expo/vector-icons'; // eslint-disable-line import/no-extraneous-dependencies
+
 import DateRangePicker from './DateRangePicker';
 import TimelineChart from './TimelineChart';
-
 import Colors from '../../constants/Colors';
 
-const Timeline = () => {
+const Timeline = ({ handleOpenDrawer }) => {
   const [showTimeline, setShowTimeline] = useState(true);
 
   return (
     <View style={styles.root}>
       <View style={styles.dateRangeContainer}>
-        <View style={styles.iconContainer} />
+        <Left>
```

Comment:
I believe `<Left>` and such elements form NB have default container sizes, I think we want to avoid those.

Based on my prev comment about controlling the DateRangePicker width here, I'd do something like:

`
<View style={[styles.dateRangeContainer, {justifyContent: 'space-between']}>
  <FILTER_ICON>
  <View style={{width: 75%}}>
    <DateRangePicker />
  </View>
  <CHEVRON_ICON>
</View>
`